### ISSUE TITLE
Light Theme: Error when deleting user

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#133]
I am attempting to delete a dummy user and encounter an error. This happens in a couple of sites (for example, graneymetaldesign.corbettresearchgroup.com and coastalclockandchime.corbettresearchgroup.com)

Example: I am trying to delete the user "Test Three" from coastalclockandchime.corbettresearchgroup.com. I am logged in as the admin (please note that I am changing the admin passwords for all sites as I go to Gk7rTsm3qPPQ. So if you are logging in as admin@example.com, please use that password.)

When I open the user Test Three and click Delete, here is the error screen I see:

![screenshot 2015-05-26 at 1 45 30 pm](https://cloud.githubusercontent.com/assets/5215893/7819088/81be20b2-03ad-11e5-9689-236e9a666ca8.png)

**Comment by meetinghouse** [comment]
Andy -- It seems I am still unable to delete any users without getting an error. Is this perhaps not rolled out yet? Is there a site where I can test it? Sorry to re-open an issue.

**Comment by acavanagh** [comment]
which site?

**Comment by meetinghouse** [comment]
http://blackburnrestoration.corbettresearchgroup.com/
and
http://coastalclockandchime.corbettresearchgroup.com/

**Comment by acavanagh** [comment]
I just deployed the new code to those sites - can you try again?

**Comment by meetinghouse** [comment]
Progress!

I am not getting the error screen anymore. However, when I try to delete the dummy user "Test Three" I get a message saying I cannot delete an admin user. I agree it's a good thing to not allow deletion of admin users. But as far as I can tell, Test Three isn't an admin user (I logged in as Test Three and I don't see the "Admin Users" link in the top menu, anyway). 

I was able to delete Test Two just fine.

Is Test Three actually an admin user?
